pkg/controller/jobs/rayjob/rayjob_webhook.go

```diff
@@ -101,7 +101,8 @@ func (w *RayJobWebhook) validateCreate(job *rayv1.RayJob) field.ErrorList {
 	var allErrors field.ErrorList
 	kueueJob := (*RayJob)(job)
 
-	if w.manageJobsWithoutQueueName || jobframework.QueueName(kueueJob) != "" || features.Enabled(features.LocalQueueDefaulting) {
+	queueName := jobframework.QueueName(kueueJob)
+	if w.manageJobsWithoutQueueName || queueName != "" || (features.Enabled(features.LocalQueueDefaulting) && queueName == "default") {
```

Comment:
This seems like a bugfix unrelated to the graduation itself. 

Would it be feasible (ideally with unit test) to extract to separate PR we may cherry-pick?

Reply:
Thanks for the review!
I just created this PR https://github.com/kubernetes-sigs/kueue/pull/5073

Reply:
Thanks, let's drop from this PR then.